src/main/kotlin/no/skatteetaten/aurora/gradle/plugins/mutators/JavaApplicationTools.kt

```diff
@@ -0,0 +1,497 @@
+                    "outputFormatter" to "json"
+                    "outputDir" to "build/dependencyUpdates"
+                    "reportfileName" to "report"
+                    "resolutionStrategy" {
+                        "componentSelection" {
+                            "all" {
+                                "boolean rejected = ['alpha', 'beta', 'pr', 'rc', 'cr', 'm', 'preview'].any { " +
+                                    "qualifier -> candidate.version ==~ /(?i).*[.-]\${qualifier}[.\\d-]*/ }"
+                                "if (rejected) { reject('Release candidate') }"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        return AuroraReport(
+            name = "plugin com.github.ben-manes.versions",
+            description = "only allow stable versions in upgrade"
+        )
+    }
+
+    fun applyDeliveryBundleConfig(bootJar: Boolean): AuroraReport = when {
+        bootJar -> {
+            with(project) {
+                plugins.apply("distribution")
+
+                withGroovyBuilder {
+                    "distributions" {
+                        "main" {
+                            "contents" {
+                                "from"("$buildDir/libs") {
+                                    "into"("lib")
+                                }
+
+                                "from"("$projectDir/src/main/dist/metadata") {
+                                    "into"("metadata")
+                                }
+                            }
+                        }
+                    }
+
+                    "distZip" {
+                        "dependsOn" to "bootJar"
+                        "archiveClassifier" to "Leveransepakke"
+                    }
+                }
+            }
+
+            AuroraReport(
+                name = "aurora.applyDeliveryBundleConfig",
+                pluginsApplied = listOf("distribution"),
+                description = "Configure Leveransepakke for bootJar"
+            )
+        }
+        else -> {
+            with(project) {
+                plugins.apply("application")
+
+                withGroovyBuilder {
+                    "distZip.classifier" to "Leveransepakke"
+                    "startScripts.enabled" to false
+                }
+            }
+
+            AuroraReport(
+                name = "aurora.applyDeliveryBundleConfig",
+                pluginsApplied = listOf("application"),
+                description = "Configure Leveransepakke"
+            )
+        }
+    }
+
+    fun applyAsciiDocPlugin(): AuroraReport {
+        project.logger.lifecycle("Apply asciiDoctor support")
+
+        with(project) {
+            withGroovyBuilder {
+                "ext.snippetsDir" to "file"("$buildDir/generated-snippets")
+
+                "asciidoctor" {
+                    "attributes([ snippets: snippetsDir, version : version ])"
```

Comment:
w: /Users/m89870/dev/test/aurora-gradle-plugin/src/main/kotlin/no/skatteetaten/aurora/gradle/plugins/mutators/JavaApplicationTools.kt: (476, 21): The expression is unused mitt bygg sier dette.